### DISCUSSION TITLE
Using Minio as a local caching layer for S3

### DISCUSSION THREAD

**Discussion by collimarco** [minio/minio#17385] [Q&A] [answered]
Is it possible to use Minio as a local caching layer (inside Kubernetes) for files already stored in the cloud in S3 (or compatible like R2)?

Is it possible to try to get a file from Minio and make Minio download the file automatically from S3 if it doesn't have it?

**Comment by harshavardhana** [comment, marked as answer]
No use MinIO directly by removing your needs from Cloud S3 or R2, you may use Tiering if you have no other way to go. https://min.io/docs/minio/linux/administration/object-management/object-lifecycle-management.html#object-transition-tiering

**Comment by jberryman** [comment]
that sounds like gateway mode, which is deprecated AFAIU ?
